Src/PerformanceCollector/Shared/Implementation/QuickPulse/Service contract/MetricPoint.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.ManagementServices.RealTimeDataProcessing.QuickPulseService
+{
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    internal struct MetricPoint
```

Comment:
We think this is best efficiency-wise right now. We will consider this option along with many bandwidth-saving ones (like using bond or other kinds of binary telemetry) in future iterations.